x-pack/plugins/fleet/server/types/rest_spec/app.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+
+export const GenerateServiceTokenRequestSchema = {
+  body: schema.object({
```

Comment:
@juliaElastic looks like it break normal service tokens creation  probably should be wrapped in a maybe
```
{
    "error": "Bad Request",
    "message": "[request body]: expected a plain object value, but found [null] instead.",
    "statusCode": 400
}

```